### ISSUE TITLE
Suggestion: set User-Agent for fetch

### ISSUE THREAD

**Issue by andypiper** [defnull/fediwall#13]
It would be a good idea to add a simple `User-Agent` header on the fetch requests sent by this app so that instance owners can identify the app if required.

**Comment by defnull** [comment]
Good idea, but not supported by chrome AFAIK -> https://bugs.chromium.org/p/chromium/issues/detail?id=571722
